bin/validate_ssa.py

```diff
@@ -37,12 +52,8 @@ def main(args):
 
 
 def _exit(code, passed, failed):
-    print("\nPassed tests")
-    print("=================")
-    print("\n".join(passed))
-    print("\nFailed tests")
-    print("=================")
-    print("\n".join(failed))
+    log(logging.INFO, "Passed tests", "\n".join(passed))
+    log(logging.INFO, "Failed tests", "\n".join(failed))
```

Comment:
If they failed, shouldn't this be an error? Might also just be better to log each success/failure in its own log statement so that you can grep for ERROR and find them.